src/cli/aspencli.py

```diff
@@ -191,6 +193,12 @@ class CliConfig:
             "verify": True,
             "oauth_api_config": default_oauth_api,
         },
+        "rdev": {
```

Comment:
The CLI won't work with rdev unfortunately -- there's two layers of authorization in front of our rdev web services that mean device auth flow doesn't work properly here. Maybe we should look into a workaround at some point :'(